l10n/en/firefox/new/desktop.ftl

```diff
@@ -37,7 +37,9 @@ firefox-desktop-download-blocks-social-trackers = Blocks social trackers
 firefox-desktop-download-os-availability = OS availability
 firefox-desktop-download-in-browser-screenshot = In-browser screenshot tool
 firefox-desktop-download-primary-password = Primary password
+# This will only be used when passed the parameter ?v=fx94 in the URL
```

Comment:
I think we should include a full example URL for clarity.

```suggestion
# This will only be used when passed the parameter ?v=fx94 in the URL. Example: https://www-dev.allizom.org/firefox/new/?v=fx94
```